chain/network/src/network_protocol/network.proto

```diff
@@ -317,6 +317,20 @@ message RoutingSyncV2 {
   bytes borsh = 1;
 }
 
+// Inter-process tracing information.
+message TraceContext {
+  enum SamplingPriority {
+    UNKNOWN = 0;
+    AutoReject = 1;
+    UserReject = 2;
+    AutoKeep = 3;
+    UserKeep = 4;
+  }
+  bytes trace_id = 1;
+  bytes span_id = 2;
```

Comment:
document that these are respectively 16B and 8B and add a link to opentelemetry documentation which defines these.